cost/kubecost/sizing/README.md

```diff
@@ -0,0 +1,42 @@
+# Kubecost Request Sizing Recommendations Policy Template
+
+## What it does
+
+The Kubecost Request Sizing Recommendations Policy Template is used to get recommendations for resizing instances controlled with Kubecost
+
+## Prerequisites
+
+This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Automation/ManagingCredentialsExternal.htm) for authenticating to datasources -- in order to apply this policy you must have a Credential registered in the system that is compatible with this policy. If there are no Credentials listed when you apply the policy, please contact your Flexera Org Admin and ask them to register a Credential that is compatible with this policy. The information below should be consulted when creating the credential(s).
```

Comment:
Can we change this "To apply this policy, you must have a Credential registered in the system that is compatible with this policy..Please contact your Flexera Org Admin if there are no Credentials listed when you apply the policy and request that they register a Credential compatible with it.Please refer to the information below when creating the credential(s)."